config/environments/production.rb

```diff
@@ -169,4 +169,8 @@
   config.relative_url_root = ENV['RELATIVE_URL_ROOT'] || "/b" if ENV['RELATIVE_URL_ROOT'] != "/"
 
   config.hosts = ENV['SAFE_HOSTS'].presence || nil
+  # Set the timeout
+  config.timeout = ENV["TIMEOUT"] || 10
```

Comment:
You could just add this to `config/application.rb`

Reply:
I'd also use more descriptive variables than just `timeoute`

Reply:
Totally agree.